### DISCUSSION TITLE
CMAKE now possible

### DISCUSSION THREAD

**Discussion by harryboo** [Freetz-NG/freetz-ng#712] [Allgemein] [answered]
Hi, means commit 9357bd00d735d5e168d26e0be0435f75e1db6cbc that now cmake can be used in freetz-ng so that getdns can be  updated?

**Comment by fda77** [comment, marked as answer]
No, its just a variable. 
Maybe its similar (https://github.com/Freetz-NG/freetz-ng/tree/master/include/meson.cross) to meson which was shortly added? Ive not read about it yet

**Reply by harryboo** [reply]
That would have been too good to be true.
I have no idea how the toolchains work and how to adapt them. Sorry.
It would be great to see cmake in freetz.

**Comment by fda77** [comment]
See latest changes and test it, i have no idea if the binary runs, or if some parameter where changed
